tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -571,7 +571,9 @@ exports[`expression.js - flow-verify 1`] = `
   value={option}
 />;
 
-<ParentComponent prop={<Child>test</Child>} />;
+<ParentComponent
+  prop={<Child>test</Child>}
+/>;
```

Comment:
The original text here was

```jsx
<ParentComponent prop={
  <Child>
    test
  </Child>
}/>;
```

This is another weird case like https://github.com/prettier/prettier/pull/4862/files#r203585057.